tutorials/source-modeling/plot_source_space_SNR.py

```diff
@@ -0,0 +1,56 @@
+# -*- coding: utf-8 -*-
+"""
+.. _tut-mne-sourcespace-snr:
+
+===============================
+Computing source space SNR
+===============================
+
+This example shows how to compute and plot source space SNR.
```

Comment:
is there a refernece paper?

Reply:
There isn't a reference for this yet. But @klankinen will update the example and that will point to the paper of Goldenholz et al. (HBM 2009).